### PR TITLE
Updates from design review

### PR DESCRIPTION
**Pull request by debugwand**

### CODE REVIEW

File: welcome-message/main.scss [hunk 1]
```diff
@@ -104,10 +102,14 @@
 	}
 
 	.n-welcome-banner__heading--minimized {
+		@include oTypographySans('s');
 		display: none;
 		@at-root .n-welcome-banner--collapsed & {
 			display: table-cell;
 		}
+		@include oGridRespondTo('M') {
+			@include oTypographySans('m');
```
Comment (keirog) [inline welcome-message/main.scss]: You could use [`oTypographySansSize('m')`](https://github.com/Financial-Times/o-typography#2-using-the-mixins) here

File: welcome-message/banner.html [hunk 2]
```diff
@@ -3,8 +3,8 @@ <h6 class="n-welcome-banner__heading n-welcome-banner__heading--minimized n-welc
 	<div class="o-expander__content o-expander__content--expanded n-welcome-banner__column n-welcome-banner__column--fill" aria-hidden="false">
 		<h6 class="n-welcome-banner__heading n-welcome-banner__column n-welcome-banner__column--m">Welcome to the new FT.com.</h6>
 		<p class="n-welcome-banner__column n-welcome-banner__column--xl n-welcome-banner__strapline">The same global insight. Faster than ever before on all your devices.</p>
-		<p class="n-welcome-banner__column n-welcome-banner__column--fill">
-			<a href="/tour" class="n-welcome-banner__button" data-trackable="tour-page" data-component="cta-take-tour"><span class="n-welcome-banner__cta--s">Take a </span>tour</a>
+		<p class="n-welcome-banner__column">
+			<a href="/tour" class="n-welcome-banner__button" data-trackable="tour-page" data-component="cta-take-tour"><span class="n-welcome-banner__cta--s">Take a </span>Tour</a>
```
Comment (keirog) [inline welcome-message/banner.html]: How come it‘s changing to `Tour`? It should either be `a tour` or `the Tour`, no?

Comment (debugwand) [inline welcome-message/banner.html]: To match new help page.

Comment (keirog) [inline welcome-message/banner.html]: 😢

**Pull request merged**